packages/events/src/router/router.ts

```diff
@@ -71,6 +77,23 @@ const publicProcedure = t.procedure
 export type AppRouter = typeof appRouter
 
 export const appRouter = router({
+  appConfig: router({
+    get: publicProcedure
+      .output(ApplicationConfigResponseSchema)
+      .query(async (options) => {
+        return getAppConfigurations(options.ctx.token)
+      }),
+    getCertificateTemplateSVGById: publicProcedure
+      .input(z.object({ id: z.string() }))
+      .output(z.string())
+      .query(async (options) => {
+        const template = await getCertificateTemplateById(
+          options.ctx.token,
+          options.input.id
+        )
+        return template
+      })
+  }),
```

Comment:
I think unless there's a compelling reason to do so, let's not bring these concerns into the `events` service. From memory, these should be available in the client's Redux store already? It's fine to read the Redux store from the new side if the information already exists there.